ir/instr.cpp

```diff
@@ -540,15 +540,15 @@ StateValue BinOp::toSMT(State &s) const {
                                     bi->non_poison));
       }
     }
-    return ty->aggregateVals(vals);
+    return ty->aggregateVals(vals, true);
```

Comment:
is true needed here?

Reply:
Yes, because `vals` does not contain padding values.

Reply:
but these don't return structures that need padding. only vectors, I think

Reply:
It can happen because sadd.overflow can take vectors as well:
```
call { <2 x i32>, <2 x i1> } @llvm.sadd.with.overflow.v2i32(..)
```